draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -36,6 +36,11 @@ export type TooltipProps = {
    * `overflow: scroll` or `overflow: hidden` element.
    */
   portalSelector?: string
+  /**
+   * Should the tooltip be visible on the first render. Useful for visual
+   * regression testing.
+   */
+  initiallyVisible?: boolean
```

Comment:
/Nitpick It feels more consistent to call this `isInitiallyVisible`, as we have Modal (has `isInitiallyOpen` prop), and FilterMenuButton (has `isInitialDropdownVisible` prop) following this convention.

Reply:
That makes sense. I've updated the variable name.